tools/nl/embeddings/build_embeddings.py

```diff
@@ -32,6 +32,8 @@
 
 FLAGS = flags.FLAGS
 
+flags.DEFINE_string('finetuned_model_gcs', '',
+                    'Existing finetuned model folder name on GCS')
```

Comment:
Should this default to `models.yaml` ?

Reply:
Hmm. Why? The run.sh script chooses the model path to provide if that flag is enabled. Not sure we should default to one or the other?